src/web/arr/trove/chart.arr

```diff
@@ -163,6 +173,17 @@ where:
 |#
 end
 
+fun string-to-stacktype(s :: String) -> StackType: 
+  doc: ```Converts ['none', 'absolute', relative', percent'] to their respective stacktype```
+  ask: 
+    | string-equal(s, 'none') then: grouped
```

Comment:
This feels weird, in a "One of these things is not like the others!" sort of way.  Why shouldn't this string be "grouped"?

Reply:
Yeah sorry about that the reason why it's "none" is because that's the string we give to Google for it to parse whether or not it's stacked or grouped ("none" vs "stacked"). I changed it to grouped on pyret's side since I thought that would make a bit more sense but the string we give google charts is still "none".

Reply:
Ok -- does this string ever appear to users of this code?  Or is it entirely internal to your implementation?  If it's internal, I don't worry about it as much.

Reply:
The string is entirely internal, externally the users input one of the enumerated stacktype: 
[grouped, absolute, relative, percent]